spaceship/lib/spaceship/connect_api/models/app.rb

```diff
@@ -116,34 +116,29 @@ def update(client: nil, attributes: nil, app_price_tier_id: nil, territory_ids:
       def fetch_live_app_info(client: nil, includes: Spaceship::ConnectAPI::AppInfo::ESSENTIAL_INCLUDES)
         client ||= Spaceship::ConnectAPI
         states = [
-          Spaceship::ConnectAPI::AppInfo::AppStoreState::READY_FOR_SALE,
-          Spaceship::ConnectAPI::AppInfo::AppStoreState::PENDING_APPLE_RELEASE,
-          Spaceship::ConnectAPI::AppInfo::AppStoreState::PENDING_DEVELOPER_RELEASE,
-          Spaceship::ConnectAPI::AppInfo::AppStoreState::PROCESSING_FOR_APP_STORE,
-          Spaceship::ConnectAPI::AppInfo::AppStoreState::IN_REVIEW,
-          Spaceship::ConnectAPI::AppInfo::AppStoreState::DEVELOPER_REMOVED_FROM_SALE
+          Spaceship::ConnectAPI::AppInfo::State::READY_FOR_DISTRIBUTION,
+          Spaceship::ConnectAPI::AppInfo::State::PENDING_RELEASE,
+          Spaceship::ConnectAPI::AppInfo::State::IN_REVIEW
         ]
 
         resp = client.get_app_infos(app_id: id, includes: includes)
         return resp.to_models.select do |model|
-          states.include?(model.app_store_state)
+          states.include?(model.state)
```

Comment:
I had expected to see `app_version_state` here. Are these 2 fields identical / duplicated in the returned data?

Also `appVersionState` doesn't appear in the fixture data. Could we refresh the fixture data as well, to make it more up to date with real life data returned from the API? It would make it easier to review the code. Thanks

Reply:
I've added `appVersionState` values to fixtures in a new commit. A fixture for `AppInfo` is not present yet and should be added to code.

And about the state.
Those fields are not identical. `app_version_state` is an attribute of `AppStoreVersion`. 
Filtering of `AppInfo` needs to be done based on `state`.
In the opan api spec, `AppInfo` doesn't have an app version state, but it has `state` with less values, e.g. there is no PENDING_APPLE_RELEASE or PENDING_DEVELOPER_RELEASE, but there is simply PENDING_RELEASE

For reference:
https://developer.apple.com/documentation/appstoreconnectapi/list_all_app_store_versions_for_an_app
https://developer.apple.com/documentation/appstoreconnectapi/appstoreversion/attributes
`appStoreState`: `ACCEPTED, DEVELOPER_REMOVED_FROM_SALE, DEVELOPER_REJECTED, IN_REVIEW, INVALID_BINARY, METADATA_REJECTED, PENDING_APPLE_RELEASE, PENDING_CONTRACT, PENDING_DEVELOPER_RELEASE, PREPARE_FOR_SUBMISSION, PREORDER_READY_FOR_SALE, PROCESSING_FOR_APP_STORE, READY_FOR_REVIEW, READY_FOR_SALE, REJECTED, REMOVED_FROM_SALE, WAITING_FOR_EXPORT_COMPLIANCE, WAITING_FOR_REVIEW, REPLACED_WITH_NEW_VERSION, NOT_APPLICABLE`

`appVersionState`: `ACCEPTED, DEVELOPER_REJECTED, IN_REVIEW, INVALID_BINARY, METADATA_REJECTED, PENDING_APPLE_RELEASE, PENDING_DEVELOPER_RELEASE, PREPARE_FOR_SUBMISSION, PROCESSING_FOR_DISTRIBUTION, READY_FOR_DISTRIBUTION, READY_FOR_REVIEW, REJECTED, REPLACED_WITH_NEW_VERSION, WAITING_FOR_EXPORT_COMPLIANCE, WAITING_FOR_REVIEW`

https://developer.apple.com/documentation/appstoreconnectapi/appinfo/attributes
`appStoreState`
`state`: `ACCEPTED, DEVELOPER_REJECTED, IN_REVIEW, PENDING_RELEASE, PREPARE_FOR_SUBMISSION, READY_FOR_DISTRIBUTION, READY_FOR_REVIEW, REJECTED, REPLACED_WITH_NEW_INFO, WAITING_FOR_REVIEW`

Reply:
I've added a fixture for `AppInfo`.